ksql-engine/src/test/java/io/confluent/ksql/function/UdfLoaderTest.java

```diff
@@ -213,6 +215,34 @@ public void shouldLoadFunctionWithStructReturnType() {
     );
   }
 
+  @Test
```

Comment:
I think there are more tests that we should add. in general it's a good idea to "run with coverage" and see if there's any branch of the code that isn't being tested and add a test for that